src/makeCalculator.js

```diff
@@ -4,7 +4,41 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    add(operand) {
+      this.result += operand;
+
+      return this;
+    },
+    subtract(operand) {
+      this.result -= operand;
+
+      return this;
+    },
+    multiply(operand) {
+      this.result *= operand;
+
+      return this;
+    },
+    divide(operand) {
+      if (operand !== 0) {
+        this.result /= operand;
+      }
```

Comment:
Consider throwing an Error when attempting to divide by zero instead of silently failing to perform the operation. This would make the behavior of the `divide` method more predictable and easier to debug in case of misuse.